lib/bolt/inventory.rb

```diff
@@ -73,20 +73,19 @@ def get_targets(targets)
       targets.map { |t| update_target(t) }
     end
 
-    # Should this be a public method?
-    def config_for(node_name)
-      data = @groups.data_for(node_name)
-      if data
-        Bolt::Util.symbolize_keys(data['config'])
-      end
-    end
-
     #### PRIVATE ####
     #
     # For debugging only now
     def groups_in(node_name)
       @groups.data_for(node_name)['groups'] || {}
     end
+    private :groups_in
+
+    def config_for(node_name)
```

Comment:
I could be wrong but I don't believe this is being called anywhere. I think you can just remove it if you refactor the tests.

Reply:
The `groups_in`? `config_for` is still used until your PR is merged.